gpiozero/output_devices.py

```diff
@@ -134,9 +134,8 @@ def toggle(self):
     @property
     def value(self):
         """
-        Returns :data:`True` if the device is currently active and
-        :data:`False` otherwise. Setting this property changes the state of the
-        device.
+        Returns 1 if the device is currently active and 0 otherwise. Setting
+        this property changes the state of the device.
```

Comment:
Does this change mean that any examples that read or write a boolean `.value` ought to be updated to read or write 0 or 1 instead?

Reply:
Yes I think they ought to be :+1: